projects/cashmere/src/lib/scroll-nav/nav/scroll-nav.component.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../sass/colors';
+
+.hc-scroll-nav-links-container {
+    margin-right: 30px;
+
+    .hc-scroll-nav-link {
+        padding: 10px 15px;
+        color: $offblack;
+        font-weight: 500;
+        background-color: $white;
+        cursor: pointer;
+
```

Comment:
Looks awesome @corykon and @Techgeekster - love it!  Only small suggestion I have is to reduce the font-size of the `hc-scroll-nav-link` to 14px (instead of 16px).  Otherwise, visually it's great and functionality works exactly as I would have expected.